tensornetwork/tests/network_operations_test.py

```diff
@@ -274,3 +274,24 @@ def test_full_graph_subgraph_dangling(backend):
   b[1] ^ c[1]
   edges = tn.get_subgraph_dangling({a, b, c})
   assert edges == {a[1], c[0]}
+
+
+def test_reduced_density(backend):
+  a = tn.Node(np.random.rand(3, 3, 3), name="A", backend=backend)
+  b = tn.Node(np.random.rand(3, 3, 3), name="B", backend=backend)
+  c = tn.Node(np.random.rand(3, 3, 3), name="C", backend=backend)
+  edges = tn.get_all_edges({a, b, c})
+
+  node_dict, edge_dict = tn.reduced_density([a[0], b[1], c[2]])
+
+  assert a[0].is_dangling() == False and \
```

Comment:
Make these separate assert lines.

Basically, never use `\`. I'm not sure why our pylint doesn't complain...

Reply:
Done.